src/ITensorNetworks/MPSTensor/MPSTensor.jl

```diff
@@ -0,0 +1,73 @@
+using ChainRulesCore
+
+struct MPSTensor
+  mps::MPS
+end
+
+function ITensors.inds(tensor::MPSTensor)
+  return siteinds(tensor.mps) == [nothing] ? () : noncommoninds(tensor.mps...)
+end
+
+function Base.:+(A::MPSTensor, B::MPSTensor)
+  if length(A.mps) == 1 && order(A.mps[1]) == 0
+    return MPSTensor(MPS([ITensor(A[] + B[])]))
+  else
+    return MPSTensor(A.mps + B.mps)
```

Comment:
Just to be clear, this is meant to actually add the MPS as states, not add them elementwise? If so, there should also be some truncation keyword arguments passed here since in general you want to do that approximately. Just adding `; kwargs...` would be sufficient, like:
```julia
function Base.:+(A::MPSTensor, B::MPSTensor; kwargs...)
  if length(A.mps) == 1 && order(A.mps[1]) == 0
    return MPSTensor(MPS([ITensor(A[] + B[])]))
  else
    return MPSTensor(+(A.mps B.mps; kwargs...))
  end
end
```

Reply:
This is actually used here: https://github.com/ITensor/ITensorNetworkAD.jl/blob/main/src/ITensorAutoHOOT/network_sum.jl#L9
does ITensors.sum allow kwargs?

Reply:
I guess we can always define another sum that takes kwargs, so if it doesn't allow kwargs also fine

Reply:
Yes, `sum(::Vector{MPS}; kwargs...)` accepts the same keyword arguments as `+(::MPS, ::MPS; kwargs...)` (`sum` just calls `+` until there is only one `MPS` left).

Reply:
Done, please take a look. It turns out that I need to redefine the ITensors.sum function to allow the kwargs we need.